apps/studio/components/interfaces/Organization/BillingSettings/BillingEmail.tsx

```diff
@@ -70,51 +103,82 @@ const BillingEmail = () => {
-                      form={formId}
-                      isSubmitting={isUpdating}
-                      hasChanges={hasChanges}
-                      handleReset={handleReset}
-                      helper={
-                        !canUpdateOrganization
-                          ? "You need additional permissions to manage this organization's settings"
-                          : undefined
-                      }
-                    />
-                  </div>
-                }
-              >
-                <FormSection>
-                  <FormSectionContent fullWidth loading={false}>
-                    <Input
-                      id="billing_email"
-                      size="small"
-                      label="Email address"
-                      type="text"
-                      disabled={!canUpdateOrganization}
-                    />
-                  </FormSectionContent>
-                </FormSection>
-              </FormPanel>
-            )
-          }}
+        <Form {...form}>
+          <form
+            id={FORM_ID}
+            onSubmit={form.handleSubmit(onUpdateOrganizationEmail)}
+            className="space-y-8"
+          >
+            <FormPanel
+              footer={
+                <div className="flex py-4 px-8">
+                  <FormActions
+                    form={FORM_ID}
+                    isSubmitting={isUpdating}
+                    hasChanges={form.formState.isDirty}
+                    handleReset={form.reset}
+                    disabled={!canUpdateOrganization}
+                    helper={
+                      !canUpdateOrganization
+                        ? 'You need additional permissions to update billing emails'
+                        : undefined
+                    }
+                  />
+                </div>
+              }
+            >
+              <FormSection>
+                <FormSectionContent fullWidth loading={loadingBillingCustomer}>
+                  <FormField
+                    control={form.control}
+                    name="billingEmail"
+                    render={({ field }) => (
+                      <FormItem>
+                        <FormLabel>Email address</FormLabel>
+                        <FormControl>
+                          <Input_Shadcn_
+                            type="email"
+                            {...field}
+                            placeholder="Email"
+                            disabled={!canUpdateOrganization}
+                          />
+                        </FormControl>
+                        <FormMessage_Shadcn_ />
+                      </FormItem>
+                    )}
+                  />
+
+                  <FormField
+                    control={form.control}
+                    name="additionalBillingEmails"
+                    render={({ field }) => (
+                      <FormItem>
+                        <FormLabel>Additional emails</FormLabel>
```

Comment:
probably good to add a description what additional emails do? will they be CCed in the same email or all of these emails will also receive the same emails as the above email address? (mainly to address potential confusion between how the addresses here differ from the above one)

Reply:
planning on adding a tooltip